scapy/arch/bpf/consts.py

```diff
@@ -12,13 +13,19 @@
 BPF_BUFFER_LENGTH = MTU
 
 # From net/bpf.h
+
+
+class bpf_program(Structure):
+    _fields_ = [("bf_len", c_uint), ("bf_insns", c_void_p)]
```

Comment:
We already have it in scapy. You should import it from scapy.arch.common

(We have
```
class bpf_program(Structure):
    """"Structure for BIOCSETF"""
    _fields_ = [("bf_len", c_uint),
                ("bf_insns", POINTER(bpf_insn))]
```